### PR TITLE
feat: Add Functions to Self Hosted

### PR DESCRIPTION
**Pull request by filipecabaco**
Adds Functions to Supabase Self Hosted

### CODE REVIEW

File: docker/docker-compose-logging.yml [hunk 1]
```diff
@@ -80,6 +80,15 @@ services:
       options:
         syslog-address: "tcp://localhost:${VECTOR_PORT}"
         tag: meta
+  function:
```
Comment (kiwicopple) [inline docker/docker-compose-logging.yml:83]:
```suggestion
  functions:
```

File: docker/docker-compose.yml [hunk 2]
```diff
@@ -241,6 +241,22 @@ services:
       PG_META_DB_USER: supabase_admin
       PG_META_DB_PASSWORD: ${POSTGRES_PASSWORD}
 
+  function:
```
Comment (kiwicopple) [inline docker/docker-compose.yml:244]:
```suggestion
  functions:
```

**Comment by reteps** [comment]
Would this allow one to make HTTP requests to self-hosted functions from postgres /the database provider?

**Review by alaister — APPROVED** [review]
LGTM!

**Pull request merged**